fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/FileObject.java

```diff
@@ -68,6 +68,12 @@ public FileObject validateType() throws FedoraException {
 		return this;
 	}
 
+	@Override
```

Comment:
Because it is inherited from the ContentObject parent class.  FileObjects don't get to have any children other than binary files (which are not presently ContentObjects), unlike the other ContentObject subclasses, but they are otherwise usable in the same ways as the others.  I could introduce another parent class where the addMember method comes in, something like ContentContainerObject, and that might be better, particularly if there is more shared functionality exclusive to container types (mainly access control comes to mind):

RepositoryObject > ContentObject > ContentContainerObject > FolderObject
RepositoryObject > ContentObject > FileObject